.gitignore

```diff
@@ -65,6 +65,7 @@ C:\\nppdf32Log\\debuglog.txt
 /config/sandbox_proxy.yml
 /config/cache_store.yml
 /config/google_experiments.yml
+/config/ttl.yml
```

Comment:
Why ttl.yml?

Reply:
@hallelujah For this https://github.com/3scale/porta/pull/574#discussion_r254237984